src/curriculum.ts

```diff
@@ -0,0 +1,17 @@
+enum nivelIngles {A1,A2,B1,B2,C1,C2};
```

Comment:
```suggestion
enum nivelIngles {
  A1 = "A1",
  A2 = "A2",
  B1 = "B1",
  B2 = "B2",
  C1 = "C1",
  C2 = "C2"
};
```
Como la lógica de negocio va a ser un scraping de los currículums, me conviene que el valor de los enum sean en forma de string para poder comparar las cadenas encontradas.